packages/client/src/MetricsPublisher.ts

```diff
@@ -4,31 +4,12 @@ import { DestroySignal } from './DestroySignal'
 import { MetricsReport } from 'streamr-network'
 import { NetworkNodeFacade, getEthereumAddressFromNodeId } from './NetworkNodeFacade'
 import { Publisher } from './publish/Publisher'
-import { ConfigInjectionToken, MetricsPeriodConfig, StrictStreamrClientConfig } from './Config'
-
-const DEFAULT_PERIODS = [ 
-    {
-        "duration": 5000,
-        "streamId": "streamr.eth/metrics/nodes/firehose/sec"
-    },
-    {
-        "duration": 60000,
-        "streamId": "streamr.eth/metrics/nodes/firehose/min"
-    },
-    {
-        "duration": 3600000,
-        "streamId": "streamr.eth/metrics/nodes/firehose/hour"
-    },
-    {
-        "duration": 86400000,
-        "streamId": "streamr.eth/metrics/nodes/firehose/day"
-    }
-]
+import { ConfigInjectionToken, MetricsPeriodConfig, STREAM_CLIENT_DEFAULTS, StrictStreamrClientConfig } from './Config'
 
 const getPeriodConfig = (rootConfig: StrictStreamrClientConfig): MetricsPeriodConfig[] => {
     switch (rootConfig.metrics) {
         case true:
-            return DEFAULT_PERIODS
+            return (STREAM_CLIENT_DEFAULTS.metrics as any).periods
```

Comment:
Why `as any`?

Reply:
The type of `STREAMR_CLIENT_DEFAULTS.metrics` is `MetricsConfig`, which could be:
- `{ periods: MetricsPeriodConfig[] }`
- or `boolean`

Reply:
Now using an explicit cast